pkg/cluster/cluster.go

```diff
@@ -496,3 +509,20 @@ func IsUnroutable(host string) bool {
 	}
 	return false
 }
+
+func makeConfig(networkType string) (*memberlist.Config, error) {
+	var mc *memberlist.Config
+
+	switch networkType {
+	case LanNetworkPeerType:
+		mc = memberlist.DefaultLANConfig()
```

Comment:
Nit: If you return early under each case, you can remove the variable declaration. The final return statement can also be removed.

Reply:
Yes, but I would actually stick to this flow - consistent with rest code, less LOCs and same readability if not better.

Reply:
yea, both option are fine to me